showcase/tests/integration/components/hds/side-nav/index-test.js

```diff
@@ -217,6 +218,9 @@ module('Integration | Component | hds/side-nav/index', function (hooks) {
     assert
       .dom('#test-side-nav-footer')
       .doesNotHaveAttribute('data-test-minimized');
+    assert
```

Comment:
should we also test that another element (`<span>`) without class `hds-side-nav-hide-when-minimized` doesn't have the `inert` attribute (in both conditions)? just in case

Reply:
added in 4e561f0